dumpgenerator.py

```diff
@@ -1058,53 +1065,85 @@ def generateImageDump(config={}, other={}, images=[], start='', session=None):
     if not os.path.isdir(imagepath):
         print 'Creating "%s" directory' % (imagepath)
         os.makedirs(imagepath)
-
+    
+    maxseconds = 100  # max seconds to wait in a single sleeping
```

Comment:
Should we put this as something that the user can configure, or should we enforce this as a hard limit?